docs/pages/blog/mui-x-v5.3.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Introducing Row Grouping Feature'
+description: After more than a year of development on the DataGrid, the first premium feature is live.
+date: 2022-01-15T00:00:00.000Z
+authors:
+  ['oliviertassinari', 'm4theushw', 'flaviendelangle', 'DanailH', 'alexfauquette']
+card: true
+---
+
+<!-- <img src="/static/blog/2021/card.png" alt="" style="width: 100%; margin-bottom: 16px;" /> -->
+
+# A step to premium plan: Row Grouping
+
+A lot of you were asking about it, and we are pleased to announce that the premium plan is on the road.
+After an incredible year fully focused on improving the DataGrid, we are moving forward by releasing a first feature of the premium plan: the row grouping which is released in v5.3.0.
```

Comment:
To avoid repetition

```suggestion
After an incredible year fully focused on improving the DataGrid, we are moving forward by releasing a first feature of the premium plan: the row grouping which is available in v5.3.0.
```

Or 

```suggestion
After an incredible year fully focused on improving the DataGrid, we are moving forward by launching the first feature of the premium plan: the row grouping which is released in v5.3.0.
```